### ISSUE TITLE
can not reproduce the result reported in the paper？

### ISSUE THREAD

**Issue by Nusselder9** [lmbxmu/RBNN#1]
We use your code to reproduce the  ResNet-20-1w1a on CIFAR. However, we can not get the **86.5%** reported in the paper.
When we set epoch as 400 epoch, the result is **85.1%**.
When we set epoch as 1000 epoch, the result is **86.1%**.

**Comment by lmbxmu** [comment]
So, what is your point ?

**Comment by lmbxmu** [comment]
> We use your code to reproduce the ResNet-20-1w1a on CIFAR. However, we can not get the **86.5%** reported in the paper.
> When we set epoch as 400 epoch, the result is **85.1%**.
> When we set epoch as 1000 epoch, the result is **86.1%**.

So, what is your point ?

**Comment by Nusselder9** [comment]
> > We use your code to reproduce the ResNet-20-1w1a on CIFAR. However, we can not get the **86.5%** reported in the paper.
> > When we set epoch as 400 epoch, the result is **85.1%**.
> > When we set epoch as 1000 epoch, the result is **86.1%**.
> 
> So, what is your point ?

Can you explain what may cause it or you just think it's common on CIFAR?

**Comment by lmbxmu** [comment]
> Can you explain what may cause it or you just think it's common on CIFAR?

First, we have explicitly provided our quantized model, training log and experimental configuration. Have you referred to it and followed the parametric settings?

Second, when asking questions, plz stress what you want. How the hell should I know your point by simply posting your own results here?

Third, our results on ResNet-20-1w1a vary from 86.3% to 86.7%. So we pick up model with accuracy of 86.5% as our paper data.  I think your data is quite weird, or at least it is beyond the limits of our experimental error. Assuming that you have followed our parametric settings, it may be related to your experimental environment such as pytorch version, running machine, etc.
